relationships/synthesis/INFRA-NR_K8S_CONTAINER-to-EXT-SERVICE.yml

```diff
@@ -0,0 +1,34 @@
+relationships:
+  - name: nrInfraK8sContainerEXTService
```

Comment:
Hi,

I think the definition below would try to relate containers to containers.
Could you double check ?

Reply:
@andre-nr, Thanks for reviewing the PR, I have updated it. Please review it again.